lib/promise.js

```diff
@@ -5,4 +5,4 @@ const promise = ({ dispatch }, next, res) =>
     ? res.then(dispatch, dispatch)
     : next(res)
 
-module.exports = curry(promise)
+exports.promise = curry(promise)
```

Comment:
Why the rename?

Reply:
To match the filename with the exported function name.  Does that end up making it less clear?

Reply:
Just curious.